uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Component/Group.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use CRM_HRContactActionsMenu_Component_GroupItem as ActionsGroupItem;
+
+/**
+ * Class CRM_HRContactActionsMenu_Component_Group
+ */
+class CRM_HRContactActionsMenu_Component_Group {
```

Comment:
Please add a description of what the class does/represents. Please also to the same for the other classes